tests/cluster/sql/test_deliverysqlspn.py

```diff
@@ -0,0 +1,15 @@
+from atc.config_master import TableConfigurator
+
+from . import extras
+from .DeliverySqlServerSpn import DeliverySqlServerSpn
+from .test_deliverysql import DeliverySqlServerTests
+
+
+class DeliverySqlServerSpnTests(DeliverySqlServerTests):
```

Comment:
Should there be an integration here? I Only see a test setUpClass, and no test?

Reply:
@JeppeBlixen  - it inhreits "DeliverySqlServerTests" and overwrites the setupclass to use the sqlserver with SPN credentials. See line 8.